src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/repository.py

```diff
@@ -25,6 +26,8 @@ def _obtain_data_from_registry(login_server, path, resultIndex, username, passwo
             )
         )
 
+        log_request(None, response.request)
```

Comment:
yes exactly. This is what the PR does. --debug already outputs the BEARER token as a part of the output. We need to see the authorization header that is being sent similar to the BEARER token to ARM.